src/parser.rs

```diff
@@ -205,6 +211,11 @@ pub fn parse_lcov(
                     });
 
                 if key.is_none() {
+                    if ignore_parsing_error {
+                        parsing_error_occurs = true;
+                        warn!("Invalid key at line {line}, ignore this record");
```

Comment:
Here and elsewhere in this file, I think it's better to move the iterator to the next EOL (included).
Just in case there is something like `DA:123SF...`